cli/build.gradle

```diff
@@ -139,16 +139,16 @@ downloadPortal {
 	onlyIfNewer true
 }
 
-downloadProductInfo {
-	src "https://releases-cdn.liferay.com/tools/workspace/.product_info.json"
+downloadReleasesInfo {
+	src "http://localhost:3000/releases.json"
```

Comment:
The releases.json should be available on a real server somewhere before we merge this.  Otherwise the tests won't be valid.

Reply:
@gamerson Thanks a lot. This is just a test url before we get the final descision for release info. Because we don't have the completely relases.json, I just put here to let you and drew to do some review. I will do other changes base on your review comments.